Sb order confirmation/purchased items widget

Tim-M-S opened the pull request:
### All changes meet the following requirements
~~[ ] Changelog entry was added~~
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io

Tim-M-S committed:
initial upload

Tim-M-S committed:
added arrow to collapsable

Tim-M-S committed:
removed old scss

Tim-M-S committed:
added paddings

Tim-M-S committed:
added translations

Tim-M-S committed:
finish

fmutschler commented on resources/lang/en/Template.properties:
```diff
@@ -468,6 +468,7 @@ orderConfirmationPaymentStatus_overpaid = "Overpaid"
 orderConfirmationPaymentStatus_partlyPaid = "Partially paid"
 orderConfirmationPaymentStatus_prepaid = "Prepaid"
 orderConfirmationPaymentStatus_unpaid = "Not paid"
+orderConfirmationProperties = "Order properties"
```
```suggestion
orderConfirmationProperties = "Order characteristics"
```

Tim-M-S replied:
done

fmutschler commented on resources/lang/en/Widget.properties:
```diff
@@ -735,3 +735,5 @@ googleMapsPlaceholderTitle = "No active map"
 ; AcceptPrivacyPolicyWidget
 acceptPrivacyPolicyLabel = "Privacy policy checkbox"
+; ItemsBoughtWidget
+itemsBoughtLabel = "Items bought"
```
```suggestion
itemsBoughtLabel = "Purchased items"
```

Tim-M-S replied:
done

Tim-M-S committed:
renamed widget
changes requested by f.mutschler

Tim-M-S committed:
# Conflicts:
#	resources/lang/de/Template.properties
#	resources/lang/de/Widget.properties
#	resources/lang/en/Template.properties
#	resources/lang/en/Widget.properties

Tim-M-S committed:
Merge branch 'feature/beta/sb_order_confirmation' into sb_order_confirmation/items_bought_widget

# Conflicts:
#	resources/js/dist/ceres-app.js.map
#	resources/js/dist/ceres.min.js
#	resources/js/dist/ceres.min.js.map

Tim-M-S committed:
Merge branch 'feature/beta/sb_order_confirmation' into sb_order_confirmation/items_bought_widget

Tim-M-S commented:
closed
